src/torchcodec/_core/_metadata.py

```diff
@@ -80,6 +81,11 @@ class VideoStreamMetadata(StreamMetadata):
     average_fps_from_header: Optional[float]
     """Averate fps of the stream, obtained from the header (float or None).
     We recommend using the ``average_fps`` attribute instead."""
+    pixel_aspect_ratio: Optional[Fraction]
+    """Pixel Aspect Ratio (PAR), also known as Sample Aspect Ratio
+    (SAR --- not to be confused with Sample Aspect Ratio, also SAR),
```

Comment:
> known as **Sample Aspect Ratio** (SAR --- not to be confused with **Sample Aspect Ratio**

Emphasis mine above, is there a typo? Both are the same.

But maybe we don't even need to mention Sample Aspect Ratio at all :thinking: ?